### PR TITLE
Typos fixed

### PR DESCRIPTION
**Pull request by sobolevn**
I am using https://github.com/codespell-project/codespell to find typos, it works pretty well!
We can even automate this check later on:
- We need to create `whitelist.txt` of words that are ignored, like `strat` 
- We need to define what files and folders to skip completely

Full example: https://github.com/dry-python/returns/blob/master/setup.cfg#L191-L194

### CODE REVIEW

**Comment by Zac-HD** [comment]
LGTM!

I also tried to automate this as part of our `lint` and `format` jobs, but it didn't *quite* work :disappointed:  
Up to you whether you'd prefer to finish doing so or just roll it back.

**Comment by sobolevn** [comment]
@Zac-HD thanks a lot for your help! I will finish it next week 👍

**Comment by sobolevn** [comment]
It took me more time than a week 😅

File: tooling/src/hypothesistooling/__main__.py [hunk 1]
```diff
@@ -63,6 +63,18 @@ def check_installed():
     don't fail to run if a previous install failed midway)."""
 
 
+@task()
```
Comment (Zac-HD) [inline tooling/src/hypothesistooling/__main__.py:66]: I don't think it's worth having this as an independent task; we only make it a function in order to call it from both `lint()` and `format()`.

File: tooling/src/hypothesistooling/__main__.py [hunk 2]
```diff
@@ -63,6 +63,18 @@ def check_installed():
...
+def codespell(*files):
+    pip_tool(
+        "codespell",
+        "--check-hidden",
+        "--check-filenames",
+        "--ignore-words=./tooling/ignore-list.txt",
+        "--skip=__pycache__,_build,.mypy_cache,.venv,.git,tlds-alpha-by-domain.txt",
```
Comment (Zac-HD) [inline tooling/src/hypothesistooling/__main__.py:73]:
```suggestion
```
These are already handled by the logic which chooses while files to format, or the comprehension in `lint()`.

**Review by Zac-HD — APPROVED** [review]
Two tiny nitpicks, but looks good to me overall - thanks for fixing this up, and feel free to merge when you think it's done.

File: tooling/src/hypothesistooling/__main__.py (hunk 2 above)
Comment (sobolevn) [reply tooling/src/hypothesistooling/__main__.py]: Actually, no. We still need this line for some reason.
I was also surprised. 

Proof (calling `codespell` without `--skip`) by `TASK=codespell ./build.sh`:
Снимок экрана 2021-03-27 в 8 51 25

File: tooling/src/hypothesistooling/__main__.py (hunk 1 above)
Comment (sobolevn) [reply tooling/src/hypothesistooling/__main__.py]: Done! 👍

File: tooling/src/hypothesistooling/__main__.py (hunk 2 above)
Comment (Zac-HD) [reply tooling/src/hypothesistooling/__main__.py]: Weird!  If that's how it works though, let's leave it as-is 🙂

**Comment by sobolevn** [comment]
Thanks a lot, @Zac-HD! 👍

**Pull request merged**